changelogs/fragments/2461-ovirt4-fix-configparser.yml

```diff
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - scripts/inventory/ovir4.py - fix configparser creation (https://github.com/ansible-collections/community.general/issues/674).
```

Comment:
```suggestion
  - ovir4 inventory script - improve configparser creation to avoid crashes for options without values (https://github.com/ansible-collections/community.general/issues/674).
```